### PR TITLE
Optimize code comments and code logs

### PR DESCRIPTION
**Pull request by JiaweiGithub**
Signed-off-by: jiawei <jiawei.liu@daocloud.io>

### CODE REVIEW

**Comment by Poorunga** [comment]
Please give a brief description of your work in PR title. For example: optimize code comments and code logs.

**Comment by JiaweiGithub** [comment]
> @JiaweiGithub Please give an accurate description of your work in PR title. For example: optimize code comments and code logs.

Already modified

File: pkg/tunnel/module.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func (t *EdgeTunnel) Shutdown() {
 	close(t.stopCh)
 }
 
-// Register register EdgeTunnel to beehive modules
+// Register EdgeTunnel to beehive modules
```
[the remark this answers is not in the document]
Comment (JiaweiGithub) [reply pkg/tunnel/module.go]: I think it's better, edgetunnel and edgeproxy.

**Pull request merged**